admin/check/check_config_inc.php

```diff
@@ -117,5 +117,15 @@
 	array( false => 'Use "limit_view_unless_threshold" instead.' )
 );
 
+# Check that 'ldap_server' is a proper URI, starting with either ldap:// or ldaps://
+$t_ldap_server = config_get_global( 'ldap_server' );
+$t_starts_with1 = substr( $t_ldap_server, 0, strlen( 'ldap://' ) ) === 'ldap://';
+$t_starts_with2 = substr( $t_ldap_server, 0, strlen( 'ldaps://' ) ) === 'ldaps://';
+check_print_test_row(
+	'"ldap_server" must not be a plain host name, it must be a proper URI',
```

Comment:
Reword to `"ldap_server" must be a valid, full LDAP URI`

Reply:
OK